### PR TITLE
DOCSP-6829: Add webview panel to display html content with bundle file

### PR DESCRIPTION
**Pull request by rayangler**
[JIRA Ticket](https://jira.mongodb.org/browse/DOCSP-6829)

**DO NOT MERGE RIGHT AWAY**

The purpose of this ticket was to create a working prototype for the webview panel to see what the previewed page will look like within VS Code. By doing this, we were able to identify components and steps that this implementation would require:
* Bundle script file (obtained by copying the file manually from local `snooty` repo)
* CSS file (for the template of the page)
* Establishing communication between VS Code and Snooty Frontend to run webpack task to create the bundle file

Moving forward, I would like to try out using `snooty` as a submodule for the extension. After running the `Snooty Preview` command on the extension, a task would execute to run `webpack` within the `snooty` submodule. This task should pass along information from the VS Code extension to `snooty`, including the page AST, to create the bundle file. The created bundle file should then be used by the webview implementation introduced in this PR to preview the page.

Let me know if this sounds like a possible approach!

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -104,9 +105,35 @@ export async function activate(context: vscode.ExtensionContext) {
...
-
     context.subscriptions.push(clickInclude);
 
+    // Command for getting page ast
+    const getPageAST: vscode.Disposable = vscode.commands.registerCommand('snooty.getPageAST', async () => {
+        const textDocument: vscode.TextDocument = vscode.window.activeTextEditor.document;
+        const fileName: string = textDocument.fileName;
+        
+        // Only valid on .txt site pages
+        if (!fileName.endsWith(".txt")) {
```
Comment (i80and) [inline src/extension.ts]: We need to lift this restriction in the not-too-far-future

File: src/webview.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  const bundlePath = path.join(
+    context.extensionPath,
+    "snooty-preview/preview",
+    "bundle.js"
+  );
+  const bundleFile = getResource(bundlePath); // TODO: Add validation through parser to see which css file to use. // For testing purposes, only support guides until we get communication between extension and frontend working
+  const cssPath = path.join(
+    context.extensionPath,
+    "snooty-preview/static/docs-tools",
+    "guides.css"
```
Comment (i80and) [inline src/webview.ts]: Not always going to be guides.css

File: src/extension.ts (hunk 1 above)
Comment (rayangler) [reply src/extension.ts]: Yup, this restriction should be removed after [DOCSP-7187](https://jira.mongodb.org/browse/DOCSP-7187) is done

File: src/webview.ts (hunk 2 above)
Comment (rayangler) [reply src/webview.ts]: This was left in for testing but will be changed in a separate PR to allow for multiple templates  ([DOCSP-7186](https://jira.mongodb.org/browse/DOCSP-7186)). Will make that PR after merging the PR for the parser

**Pull request closed without merging**